general/forall.hpp

```diff
@@ -361,13 +336,7 @@ inline void ForallWrap(const bool use_dev, const int N,
 #if defined(MFEM_USE_RAJA) && defined(RAJA_ENABLE_OPENMP)
    // Handle all allowed OpenMP backends except Backend::OMP
    if (DIM == 1 && Device::Allows(Backend::OMP_MASK & ~Backend::OMP))
-   { return RajaOmpWrap1D(N, h_body); }
-
-   if (DIM == 2 && Device::Allows(Backend::OMP_MASK & ~Backend::OMP))
-   { return RajaOmpWrap2D(N, h_body, X, Y, Z); }
-
-   if (DIM == 3 && Device::Allows(Backend::OMP_MASK & ~Backend::OMP))
-   { return RajaOmpWrap3D(N, h_body, X, Y, Z); }
```

Comment:
@camierjs , I simplified the OpenMP back end since multiple iteration spaces are not needed.

Reply:
That sounds good, thanks again @artv3 !